modules/schematics/src/store/index.spec.ts

```diff
@@ -18,7 +18,7 @@ describe('Store Schematic', () => {
   );
   const defaultOptions: StoreOptions = {
     name: 'foo',
-    project: 'bar',
```

Comment:
Don't modify existing defaults. Add an additional test that provides an empty project